modulemd/tests/test-modulemd-modulestream.c

```diff
@@ -2179,6 +2236,20 @@ main (int argc, char *argv[])
               module_stream_v2_test_profiles,
               NULL);
 
+  g_test_add ("/modulemd/v2/modulestream/v1/rpm_api",
```

Comment:
Please convert this to `g_test_add_func()` [see the GLib manual on how to use that] since we are going to replace it in the whole codebase with that anyway for simplicity.

Reply:
We will do that in a separate request later. I'm going to merge this as-is.